.github/workflows/pytest-coverage.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install dependencies, create coverage tests and run Pytest Coverage Comment
+# For more information see: https://github.com/MishaKav/pytest-coverage-comment/
+name: pytest-coverage-comment
```

Comment:
Could we use https://github.com/devopshq/artifactory-cleanup/blob/16254bb8848fcff47d4d9b35c0bd7a4ef01b053e/.github/workflows/release.yml and run pytest after ` Install dependencies` step? 
If `release` name confuses, let's rename it to `build.yaml` to show that it's not only about releases

It'd be great to run tests before release!

Reply:
Updated the workflow. Coverage is now not included anymore.

For this good enough. Long-term you should think about adding coverage (not in this MR)